starlark/testdata/function.star

```diff
@@ -321,3 +321,9 @@ b(1, y=2, )
 #c(1, *[], )
 #d(1, *[], z=None, )
 #e(1, *[], z=None, *{}, )
+
+---
+# Unpack provides spell check for argument names.
+load("assert.star", "assert")
+
+assert.fails(lambda: min([], keg=1), ".+did you mean key\\?")
```

Comment:
FYI, the triple-# syntax below means "implicitly assert that execution fails with an error that matches the regexp", so you don't need the assert package. But what you have here is fine too.

```
---
# Unpack provides spell check for argument names.
min([], keg=1) ### `did you mean key\?`
```